geomet_data_registry/tileindex/elasticsearch_.py

```diff
@@ -240,6 +240,58 @@ def add(self, identifier, data):
 
         return status_code
 
+    def bulk_add(self, data):
+        """
+        Add many items to the tileindex
+
+        :param data: GeoJSON dict
+
+        :returns: list of dict {layer_id: HTTP status code}
+        """
+
+        status_code = None
+
+        LOGGER.debug('Starting bulk add')
+        try:
+            # FIXME: refresh='wait_for' essentially blocks further processing
```

Comment:
Do we need this `FIXME` here?

Reply:
We can delete the FIXME. Sorry, should've done this in the pipeline MR.

Reply:
fix in merge request, marking this as resolved.